software/deploy/deploy.go

```diff
@@ -427,3 +433,27 @@ func DagsOnlyDeploy(houstonClient houston.ClientInterface, appConfig *houston.Ap
 	}
 	return fileutil.UploadFile(&uploadFileArgs)
 }
+
+func UpdateDeploymentImage(houstonClient houston.ClientInterface, deploymentID, wsID, runtimeVersion, imageName string) error {
+	if runtimeVersion == "" {
+		return ErrNoRuntimeVersionPassed
+	}
+	if imageName == "" {
```

Comment:
nit: Should we validate that the image being passed is built using the same runtime version as the one being passed on as user input? Or does the Houston API do that for us?